web/src/features/charts/bar-breakdown/BarBreakdownChart.tsx

```diff
@@ -38,6 +38,13 @@ function BarBreakdownChart() {
     y: number;
   } | null>(null);
 
+  const [headerHeight, setHeaderHeight] = useState<number>(0);
+
+  useEffect(() => {
+    const height = document.querySelectorAll('header')[0].offsetHeight;
+    setHeaderHeight(height * 1.1);
+  }, [window.innerWidth, window.innerHeight]);
+
```

Comment:
```suggestion
  const [headerHeight, setHeaderHeight] = useState<number>(0);

  useEffect(() => {
    const height = document.querySelector('header').offsetHeight;
    setHeaderHeight(height * 1.1);
  }, [window.innerWidth, window.innerHeight]);

```
Since you are grabbing the first one anyhow let's just use querySelector() instead querySelectorAll().